PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/configuration/AzureConfigurations.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.intellij.configuration;
+
+import com.intellij.openapi.components.PersistentStateComponent;
+import com.intellij.openapi.components.ServiceManager;
+import com.intellij.openapi.components.State;
+import com.intellij.openapi.components.Storage;
+import com.intellij.util.xmlb.XmlSerializerUtil;
+import com.intellij.util.xmlb.annotations.MapAnnotation;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+
+@State(name = "Azure Toolkit for IntelliJ", storages = {@Storage("azure-data.xml")})
+public class AzureConfigurations implements PersistentStateComponent<AzureConfigurations.AzureConfigurationData> {
```

Comment:
How about to rename it to AzureToolkitSettings?
It belongs to File -> Settings ..., I feel "Settings" is better than Configuration (It sounds a "Run Configuration" a bit).

Reply:
There is already a com.microsoft.intellij.AzureSettings. can we merge these configurations into it ?

Reply:
rename @State name, file name